### PR TITLE
create namespace API

### PR DESCRIPTION
**Pull request by v-shant**
cloud functions CRUD operations

### CODE REVIEW

File: specification/resources/serverless/responses/namespace_created.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+description: The response will be a JSON object with a key called `namespace`.
+  The value of this will be an object that contains the standard attributes
+  associated with a namespace.
+
+content:
```
Comment (andrewsomething) [inline specification/resources/serverless/responses/namespace_created.yml:5]: You can use the shared ratelimit-limit components here:

```suggestion
headers:
  ratelimit-limit:
    $ref: '../../../shared/headers.yml#/ratelimit-limit'
  ratelimit-remaining:
    $ref: '../../../shared/headers.yml#/ratelimit-remaining'
  ratelimit-reset:
    $ref: '../../../shared/headers.yml#/ratelimit-reset'

content:
```

File: specification/resources/serverless/models/namespace_info.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    description: Namespace label.
+
+  region:
+    type: string
+    example: nyc
+    description: Region in which namespace created.
+
+  login_token:
+    type: string
+    example: null
```
Comment (andrewsomething) [inline specification/resources/serverless/models/namespace_info.yml:26]: This is causing a linting error as the example's type isn't a string. If `null` is a valid response, than you can add `nullable: true` If not, just add a random string to use as an example.

**Review by andrewsomething — CHANGES_REQUESTED** [review]

**Comment by andrewsomething** [comment]
!deploy

File: specification/DigitalOcean-public.v2.yaml [hunk 3]
```diff
@@ -388,6 +388,9 @@ tags:
   - name: Regions
     description: Provides information about DigitalOcean data center regions.
 
+  - name: Serverless
+    description: Provides methods for CRUD operations for namespaces.
```
Comment (piyush0101) [inline specification/DigitalOcean-public.v2.yaml:392]: nit: should this say "Provides methods for CRUD operations for function namespaces"

**Comment by scotchneat** [comment]
!delpoy

**Comment by ChiefMateStarbuck** [comment]
!deploy

**Comment by andrewsomething** [comment]
Converting this to a draft for now to make sure it is not inadvertently merged following our conversation last week.

**Pull request closed without merging**